.github/actions/run-on-image/action.yaml

```diff
@@ -45,6 +45,7 @@ runs:
     # `--env npm_config_cache` overrides the default `/.npm` to avoid permission issues.
     - run: >-
         docker run
+        ${{inputs.use_published_image == 'true' && '--pull=always' || ''}}
```

Comment:
not a bug, but worth noting an empty space is added when inputs.use_published_image is false